lib/pipeline/webpack.config.js

```diff
@@ -0,0 +1,176 @@
+} catch (e) {
+  versions = [];
+}
+
+const entry = Object.keys(embarkAssets)
+  .filter(key => key.match(/\.js?$/))
+  .reduce((obj, key) => {
+    // webpack entry paths should start with './' if they're relative to the
+    // webpack context; embark.json "app" keys correspond to lists of .js
+    // source paths relative to the top-level dapp dir and may be missing the
+    // leading './'
+    obj[key] = embarkAssets[key]
+      .map(file => {
+        let file_path = file.path;
+        if (!file.path.match(/^\.\//)) {
+          file_path = './' + file_path;
+        }
+        return file_path;
+      });
+    return obj;
+  }, {});
+
+function resolve(pkgName) {
+  if (Array.isArray(pkgName)) {
+    const _pkgName = pkgName[0];
+    pkgName[0] = require.resolve(_pkgName);
+    return pkgName;
+  }
+  return require.resolve(pkgName);
+}
+
+// base config
+// -----------------------------------------------------------------------------
+
+const base = {
+  context: dappPath,
+  entry: entry,
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: [{loader: 'style-loader'}, {loader: 'css-loader'}]
+      },
+      {
+        test: /\.scss$/,
+        use: [{loader: 'style-loader'}, {loader: 'css-loader'}]
+      },
+      {
+        test: /\.(png|woff|woff2|eot|ttf|svg)$/,
+        loader: 'url-loader?limit=100000'
+      },
+      {
+        test: /\.jsx?$/,
+        loader: 'babel-loader',
+        exclude: /(node_modules|bower_components|\.embark[\\/]versions)/,
+        options: {
+          plugins: [
+            [
+              'babel-plugin-module-resolver', {
+                'alias': embarkAliases
+              }
+            ],
+            [
+              '@babel/plugin-transform-runtime', {
+                corejs: 2,
+                useESModules: true
+              }
+            ]
+          ].map(resolve),
+          presets: [
+            [
+              '@babel/preset-env', {
+                modules: false,
+                targets: {
+                  browsers: ['last 1 version', 'not dead', '> 0.2%']
+                }
+              }
+            ],
+            '@babel/preset-react'
+          ].map(resolve)
+        }
+      }
+    ]
+  },
+  output: {
+    filename: (chunkData) => chunkData.chunk.name,
+    // globalObject workaround for node-compatible UMD builds with webpack 4
+    // see: https://github.com/webpack/webpack/issues/6522#issuecomment-371120689
+    globalObject: 'typeof self !== \'undefined\' ? self : this',
+    libraryTarget: 'umd',
+    path: buildDir
+  },
+  plugins: [new HardSourceWebpackPlugin()],
+  // profiling and generating verbose stats increases build time; if stats
+  // are generated embark will write the output to:
+  //   path.join(dappPath, '.embark/stats.[json,report]')
+  // to visualize the stats info in a browser run:
+  //   npx webpack-bundle-analyzer <dappPath>/.embark/stats.json
```

Comment:
The first error is strange because in the revised pipeline (this branch) webpack no longer writes to `.embark`.

The second error results from `./.embark/stats.report` — it should be `./.embark/stats.json`. The `webpack-bundle-analyzer` only knows how to process the JSON version of the stats.